PBSPro/pbspro
# Re-enable the PBS_alps_inventory_check hook on Cray X* systems

#### Bug Description
* The commit for "Don't enable PBS_translate_mpp hook on Cray X* series by default" accidentally also disabled the PBS_alps_inventory_check hook.   
* To verify the bug is fixed: 
install PBS on a Cray X* series
Start PBS
use qmgr -c "list pbshook" to verify that the PBS_alps_inventory_check shows "enabled = true"

I cannot attach test logs because the logs contain proprietary information.

#### Affected Platform(s)
* Platform: Cray X* series

#### Cause / Analysis / Design
* Accidentally removed too much Cray X* series specialized code

#### Solution Description
* Added code back into pbs_postinstall.in to enable the PBS_alps_inventory_check hook for Cray X* series systems.

#### Checklist:





__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__
